src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
It is possible, but if we leave it as enum it gives more checks for version differences. As rust would complain about non-exaustive enums, when enum on rust side doesn't implement everything fully. This is why I had to add missing codec ids here: https://github.com/meh/rust-ffmpeg/pull/81/files#diff-76fddcc9603872c4b31cebbaae65aedbR465